test/unit/spec/MfaVerify_spec.js

```diff
@@ -1034,6 +1034,34 @@ function (Okta,
             });
           });
         });
+        itp('posts resend if send code button is clicked second time', function () {
+          Util.speedUpPolling();
+
+          return setupSMS().then(function (test) {
+            $.ajax.calls.reset();
+            test.setNextResponse(resChallengeSms);
+            test.form.smsSendCode().click();
+            return tick(test);
+          })
+          .then(function (test) {
+            // One more tick to give it time for become enable
+            return tick(test);
```

Comment:
why do we need another tick? We should not need that, or maybe we should wait for something else to be done (instead of a tick, we could try adding a spy and wait for it, or something similar (we have changed some tests already to avoid tick)